python/wheelbuilder/Dockerfile

```diff
@@ -0,0 +1,98 @@
+FROM quay.io/pypa/manylinux1_x86_64
```

Comment:
Didn't know about this one. What's special about it?

Reply:
It's provided by the "Python Packaging Authority" (pypa) and come pre-prepared as a toolchain to create "manylinux1" wheels.  This means it's an ancient CentOS version so that all newer linuxes are supported but has some niceties like more modern compilers are available already, all supported versions of python are available etc.  And finally it comes with the tools to build a manylinux1 wheel which includes tooling to re-write .so files to add RPATH in the with $ORIGIN/.libs/ and to then also bundle the dependent .so files like this.
I.e. it's a nice base to use to compile on if you want to support all linuxes even though it's primary purpose was python wheels, but it meant they had to address the compiling bits too.  I'll probably also use this base to build a standalone deltachat.so from.